src/firmware/boards/shared/legacy_freertos/vendor/portable/RVDS/ARM_CA9/port.c

```diff
@@ -75,23 +75,23 @@
 #include "task.h"
 
 #ifndef configINTERRUPT_CONTROLLER_BASE_ADDRESS
-	#error configINTERRUPT_CONTROLLER_BASE_ADDRESS must be defined.  See http://www.freertos.org/Using-FreeRTOS-on-Cortex-A-Embedded-Processors.html
+	#error configINTERRUPT_CONTROLLER_BASE_ADDRESS must be defined.  See http://www.legacy_freertos.org/Using-FreeRTOS-on-Cortex-A-Embedded-Processors.html
```

Comment:
Also here

Reply:
Reverted all changes in `legacy_freertos`